src/Mount.cpp

```diff
@@ -107,19 +67,17 @@ const float siderealDegreesInHour = 14.95904348958;
 // CTOR
 //
 /////////////////////////////////
-Mount::Mount(float stepsPerRADegree, float stepsPerDECDegree, LcdMenu* lcdMenu) {
-   _instance = this;
-   
-  #if RA_DRIVER_TYPE != DRIVER_TYPE_ULN2003
-  _stepsPerRADegree = stepsPerRADegree * SET_MICROSTEPPING; // hier
-  #else
-  _stepsPerRADegree = stepsPerRADegree;
-  #endif
-  #if DEC_DRIVER_TYPE != DRIVER_TYPE_ULN2003
-  _stepsPerDECDegree = stepsPerDECDegree * DEC_SLEW_MICROSTEPPING;
-  #else
-  _stepsPerDECDegree = stepsPerDECDegree;
+Mount::Mount(LcdMenu* lcdMenu) :
+  _stepsPerRADegree(RA_STEPS_PER_DEGREE),   // u-steps per degree when slewing
```

Comment:
its not always microsteps ... if the driver is set to full step mode it would be just steps. also the ULN2003 does not support microstepping

Reply:
Well, I just labelled it microstepping to differentiate from full steps. Would be more correct to state "includes microstepping when microstepping enabled". I just consider half steps as large microsteps and full-steps as the degenerate unity case :-). This is also consistent with the datasheets for the non-ULN2003 drivers. If we use engineering units for the HAL drivers then a lot of this will become clearer.